packages/design-system/src/components/Icon/SizedIcon.tsx

```diff
@@ -18,6 +18,11 @@ const SizedIcon = React.forwardRef(
 	) => {
 		const numericSize = getNumericSize(size);
 		const fullName = size ? `${name}:${size}` : name;
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Helps debugging. Since on some of the TS implementations, a `string` can reach this component, this log will help point out why an icon could not be displayed.